compiler-rt/test/asan/TestCases/strcpy-overlap.cpp

```diff
@@ -36,9 +36,9 @@ __attribute__((noinline)) void bad_function() {
   char buffer[] = "hello";
   // CHECK: strcpy-param-overlap: memory ranges
   // CHECK: [{{0x.*,[ ]*0x.*}}) and [{{0x.*,[ ]*0x.*}}) overlap
-  // CHECK: {{#0 0x.* in .*strcpy}}
-  // CHECK: {{#1 0x.* in bad_function.*strcpy-overlap.cpp:}}[[@LINE+2]]
-  // CHECK: {{#2 0x.* in main .*strcpy-overlap.cpp:}}[[@LINE+5]]
+  // CHECK: {{#[0-9]+ 0x.* in .*strcpy .*.cpp}}
```

Comment:
We are seeing a failure in this test in our internal builder due to the line looking like this:
```
#0 0x55ec8a9ad17c in strcpy.part.0 /home/jenkins/compiler-rt/lib/asan/asan_interceptors.cpp:566:5
```
Previously the match would end at `strcpy`, but now the match fails because you are expecting a space immediately following the `strcopy` which isn't the case on our internal builder.

Reply:
That's different from my output. Thanks for fixing that :)